docs/_components/code-formatting.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Code Formatting
```

Comment:
According to [the Content Guide](https://content-guide.18f.gov/capitalization/#headings), headlines, page titles, subheads and similar content should follow sentence case. 

(I think sentence-case heading look a little weird personally, but those are just my tastes! I think it’s important to be consistent.)